src/DependencyInjection/Configuration.php

```diff
@@ -117,4 +117,21 @@ public function getConfigTreeBuilder()
 
         return $treeBuilder;
     }
+
+    protected function shouldMergeConfig()
+    {
+        # works with symfony/dotenv
+        if (isset($_ENV['AWS_MERGE_CONFIG'])) {
+            return $_ENV['AWS_MERGE_CONFIG'];
+        }
+
+        # works with case-insensitive names on windows and doesn't work with symfony/dotenv
+        $mergeConfig = getenv('AWS_MERGE_CONFIG');
+
+        if ($mergeConfig) {
+            @trigger_error('Since aws/aws-sdk-php-symfony 2.5.0: Support for case-insensitive AWS_MERGE_CONFIG is deprecated and will be removed in 3.0.0', \E_USER_DEPRECATED);
```

Comment:
Can you split this into a few lines to conform to PSR line limit standards?  I know that standard is a topic of debate, but it's something we follow in the PHP SDK.
example:
```
@trigger_error(
    'Since aws/aws-sdk-php-symfony 2.5.0: Support for case-insensitive'
    . ' AWS_MERGE_CONFIG is deprecated and will be removed in 3.0.0', 
    \E_USER_DEPRECATED
);
```